package.json

```diff
@@ -61,13 +64,18 @@
     "@typescript-eslint/parser": "5.55.0",
     "@vitejs/plugin-react": "3.1.0",
     "auto": "10.46.0",
+    "axios": "1.4.0",
+    "chalk": "4.1.2",
     "chromatic": "6.17.3",
     "concurrently": "8.0.1",
+    "dotenv": "16.3.1",
     "eslint": "8.36.0",
     "eslint-plugin-jest": "27.2.1",
     "eslint-plugin-storybook": "0.6.12",
+    "figma-api-exporter": "0.0.2",
     "figma-js": "1.16.0",
     "figma-transformer": "2.1.0",
```

Comment:
Maybe these are supposed to be devDependencies as well?

Reply:
I believe that's moved now.